glean-core/ios/Glean/Dispatchers.swift

```diff
@@ -158,6 +158,9 @@ class Dispatchers {
         // `waitUntilFinished` parameter since this is a serial queue and any newly queued tasks
         // should execute after the `preInitOperations` that are being added here. For tests, we
         // need to await all of the tasks to finish execution, so we set this to true.
+
+        // TODO 1591094: Android has a timeout of 5 seconds for running preinit tasks.
```

Comment:
swiftfmt formats this to `// TODO: 1591094: ...`

A better format would be: `// TODO(bug 1591094)` and then swiftfmt is happy as well

Reply:
Ah -- I thought swiftlint was just complaining about having a `TODO` at all.  (The output in CircleCI isn't terribly helpful -- I wonder if there's something we can do there...)

Reply:
Unfortunately the output is awful indeed and it is `swiftformat` that is complaining.
I'll whip something up.